<?php

try {
    // генерация исключений
    if(rand(0, 1)) {
        throw new Exception();
    }
}
catch(Exception $exp) {
    // блок обработки
    // если было исключение
    exit("Произошла исключительная ситуация");
}

echo "Штатная работа скрипта";

?>